server/app/modules/SecurityModule.java

```diff
@@ -145,53 +152,134 @@ protected FakeAdminClient fakeAdminClient(ProfileFactory profileFactory) {
     return new FakeAdminClient(profileFactory, this.configuration);
   }
 
+  // CiviForm uses HTTP basic auth for authenticating API calls. The username in the basic auth
+  // credentials is the API
```

Comment:
[nit] this can be on 2 lines instead of 3

Reply:
done